doc/import_external_docs.ps1

```diff
@@ -12,7 +12,7 @@ $external_docs = @{
     "uno.toolkit.ui"     = "a0a15f56d1ba4f496a42a678ff981b41bde6396a"
     "uno.check"          = "baf57490d5cdcb7fd209db0945615c75dcb8accb"
     "uno.xamlmerge.task" = "7e8ffef206e87dfea90c53805c45e93a7d8c0b46"
-    "figma-docs"         = "9ee95d5b3627657d6e818d79e9d6e6019fc53f5b"
+    "figma-docs"         = "abb770f1c853e9872d3826699d74107a5deb6538"
```

Comment:
You should order them alphabetically, it would be more readable

```suggestion
    "figma-docs"         = "0011223334555666777778888999aabbbcdddeef"
```